ccm_web/package.json

```diff
@@ -84,7 +86,7 @@
     "@types/passport": "1.0.16",
     "@types/passport-jwt": "4.0.1",
     "@types/react": "17.0.47",
```

Comment:
I see the current version of `@types/react:18.2.64` and this will go  tandem with `react` upgrade. is there any reason why this was not updated?

Reply:
I think I had it before but I may have lost it trying to merge with the new branch. Just made a commit to update it